src/test/java/com/diffmin/AppTest.java

```diff
@@ -131,7 +133,7 @@ public String toString() {
         }
     }
 
-    /** Provides test sources for scenarios where only delete patches are applied. */
+    /** Provides test sources for scenarios where delete and update patches are applied. */
```

Comment:
This change is unrelated to this PR but I still made it because it was too minor to be worthy of a separate PR.